app/src/main/java/com/onewelcome/showcaseapp/navigation/BottomNavigationBar.kt

```diff
@@ -42,11 +42,9 @@ fun BottomNavigationBar() {
             },
             onClick = {
               navController.navigate(navigationItem.route) {
-                popUpTo(navController.graph.findStartDestination().id) {
+                popUpTo(navController.graph.startDestinationId) {
```

Comment:
For some reason this doesn't work when you're on details page and click on the bottom button to navigate back to the root of the graph. I tried few solutions but to no avail, maybe you'll have some nice idea how to fix this without massive overhead.

Reply:
I added `launchSingleTop`, this should prevent from having multiple instances of the same destination as currently `popUpTo` was removing it to the first occurence.

Reply:
Now there's a problem with `Home` stack not "remembering" that it was on the details page. I think this can be fixed later on, it's not necessary to fix it as part of this PR.

Reply:
I added multiple backstacks and it seems to work. So there's root backstack to controll which screen from bottom navbar should be shown (home or info) and I added sub-backstack for HomeScreen that controlls moving around features. For info screen I didn't add navigation as I'm not sure it will be needed. We can always easily add it later.